### ISSUE TITLE
About Loss for test_mask

### ISSUE THREAD

**Issue by XXXHUA** [waittim/graph-fraud-detection#3]
hi, for the calculation of loss, l = loss(pred, labels), it seems to use the test_mask label for calculate loss , which is reasonable?

**Comment by waittim** [comment]
Good catch — thanks for asking. Loss should only be computed on the training (or train+val) mask, not `test_mask`. If the code currently uses test labels for training loss, that’s a bug/misleading. I’ll double-check the notebook and clarify the intended split. Could you point to the exact cell/line you saw?
